### PR TITLE
fix crash with malformed params

### PR DESCRIPTION
**Pull request by ezheidtmann**
fixes https://github.com/ridewithgps/ridewithgps/issues/15605

now we return an empty array if the params are missing or blank

```
09:37 $ curl http://localhost:4002/get_eles
[]✔ ~/rwgps/humps [master|✚ 1…1⚑ 2] 
09:41 $ curl http://localhost:4002/get_eles
[]✔ ~/rwgps/humps [master|✚ 1…1⚑ 2] 
09:41 $ curl 'http://localhost:4002/get_eles?lats=&lngs='
[]✔ ~/rwgps/humps [master|✚ 1…1⚑ 2] 
09:41 $ curl 'http://localhost:4002/get_eles?lats=34&lngs='
[]✔ ~/rwgps/humps [master|✚ 1…1⚑ 2] 
09:41 $ curl 'http://localhost:4002/get_eles?lats=34&lngs=32'
[-9999]✔ ~/rwgps/humps [master|✚ 1…1⚑ 2] 
09:41 $ curl 'http://localhost:4002/get_eles?lats=34&lngs=32,34'
[-9999,-9999]✔ ~/rwgps/humps [master|✚ 1…1⚑ 2] 
09:41 $ curl 'http://localhost:4002/get_eles?lats=34&lngs=32,34,32342'
[-9999,-9999,-9999]✔ ~/rwgps/humps [master|✚ 1…1⚑ 2] 
```

### CODE REVIEW

File: server/hump_server.rb [hunk 1]
```diff
@@ -10,6 +10,7 @@ class HumpServer < Sinatra::Base
     register Sinatra::Cors
     set :allow_origin, "*"
     set :allow_methods, "GET, HEAD, POST"
+    set :allow_headers, "content-type,x-requested-with"
```
Comment (abealsaleh) [inline server/hump_server.rb:13]: is this related to the fix?

Comment (ezheidtmann) [reply server/hump_server.rb:13]: no, sry, i didn't mean to include it. it would be nice though, for silly reasons

**Review by abealsaleh — APPROVED** [review]
